### PR TITLE
Include DevC-Islamabad

### PR DESCRIPTION
**Pull request by MianJawadAhmad**
#### What does this PR do?
I Include DevC-Islamabad in the list

#### Any relevant issues this PR addresses? (optional)
Not it all

#<NUMBER>

#### Any background context you want to add?
no

### CODE REVIEW

File: DevCGlobalDirectory.md [hunk 1]
```diff
@@ -330,6 +330,20 @@
 </details>
 
 
+## Pakistaa
```
Comment (jordanliu) [inline DevCGlobalDirectory.md:333]: Should be Pakistan?

**Comment by jordanliu** [comment]
Also, to note that this change was made in PR #88

File: DevCGlobalDirectory.md (hunk 1 above)
Comment (MianJawadAhmad) [reply DevCGlobalDirectory.md:333]: It will have its own header. :)

**Comment by saranshkataria** [comment]
Since this is already in #88, closing this one out.

**Pull request closed without merging**